torchao/experimental/ops/mps/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+
+python ../../kernels/mps/codegen/gen_metal_shader_lib.py
```

Comment:
just add one line comment on what this does

Reply:
BTW in this script, why depend on yaml file for specification. Can you not just assume some pattern and include those metal files? Why add another step for someone to find their kernel needs extra step to be available to runtime

Reply:
Because once I start optimizing the shaders, I will very quickly be in a situation where I will have multiple shaders per op, meaning, multiple versions. We want to benchmark them, and compare them. They should coexist. The yaml is the way to point at the best one, the "production one". We might end up in a place where one version is best for one platform/device, while another is best for another. If we get to that point, we can enrich the yaml to accommodate for those options. In the meantime, having this design allows me to very easily swap one version for another, and do comparisons.

Reply:
ok sounds good